app/models/user.rb

```diff
@@ -0,0 +1,40 @@
+require 'bcrypt'
+require 'dm-validations'
+
+class User
+
+  include DataMapper::Resource
+
+  property :id, Serial
+  property :name, String, required: true
+  property :username, String, unique: true, required: true
+  property :email, String, unique: true, format: :email_address, required: true,
+           :messages => {
+              :presence => 'Please enter an email address',
+              :is_unique => 'That email address is already registered',
+              :format => 'Please enter a valid email address'
+            }
+  property :password_digest, Text
+
+  has n, :peeps, :through => Resource
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)